src/main/resources/reference.conf

```diff
@@ -301,5 +301,9 @@ jicofo {
       // Only disable this if your xmpp connection is on loopback!
       use-tls = true
     }
+
+    // The list of domains with trusted services. Only members logged in to these domains can declare themselves to be
+    // Jibri instances.
+    trusted-domains = []
```

Comment:
Can we change it to default to `recorder .${xmpp.domain}` so we avoid breaking all deployments on next update?